### PR TITLE
feat: add --no-upgrade flag

### PR DESCRIPTION
**Pull request by zlalvani**
This is useful in cases where you use a different tool to perform the upgrade, or you did it manually

### CODE REVIEW

File: apps/cli/src/App.tsx [hunk 1]
```diff
@@ -254,6 +255,7 @@ const App = (props: {
         "-p",
         `${port}`,
         ...(token ? ["-t", token] : []),
+        ...(props.upgrade ? [] : ["-n"]),
```
Comment (noqcks) [inline apps/cli/src/App.tsx:258]: does upgrade need to come before token in the args?

File: apps/cli/src/index.tsx [hunk 2]
```diff
@@ -40,11 +40,12 @@ const command = program
     (val) => parseInt(val),
     3000,
   )
+  .option("-n, --no-upgrade", "skip applying the upgrade")
   .option("-s, --simple", "simple mode")
   .option("-i, --ipc", "run in ipc mode")
   .parse();
 
-const { model, language, port, ipc, simple, token } = command.opts();
+const { model, language, port, ipc, simple, token, upgrade } = command.opts();
```
Comment (noqcks) [inline apps/cli/src/index.tsx:48]: wouldn't it be noUpgrade or something?

File: apps/cli/src/App.tsx (hunk 1 above)
Comment (zlalvani) [reply apps/cli/src/App.tsx:258]: they're flags so it shouldn't matter

File: apps/cli/src/index.tsx (hunk 2 above)
Comment (zlalvani) [reply apps/cli/src/index.tsx:48]: commander is smart enough to handle this automatically

Comment (noqcks) [reply apps/cli/src/index.tsx:48]: insane

**Review by noqcks — APPROVED** [review]

**Review by noqcks — APPROVED** [review]

File: apps/cli/src/index.tsx (hunk 2 above)
Comment (zlalvani) [reply apps/cli/src/index.tsx:48]: the types work automatically too (it parses the string)

**Pull request merged**